docs/guide/connecting.asciidoc

```diff
@@ -15,22 +15,24 @@ page after you've created a cluster (look in the top-left if you're in Kibana).
 
 We recommend using a Cloud ID whenever possible because your client will be 
 automatically configured for optimal use with Elastic Cloud including HTTPS and 
-HTTP compression.
+HTTP compression; as well as the API key rather than
+the alternative `basic_auth=(user, password)` option.
```

Comment:
I believe this is orthogonal, you can use a Cloud ID with basic auth or API keys. (Cloud ID is not a secret, it's only an URL.)